test/e2e/dra/deploy.go

```diff
@@ -428,7 +397,7 @@ func (d *Driver) SetUp(nodes *Nodes, resources Resources, devicesPerNode ...map[
 	})
 
 	// Run registrar and plugin for each of the pods.
-	for i, pod := range pods.Items {
+	for _, pod := range pods.Items {
```

Comment:
You can remove the following three lines, loop variables are distinct for each loop iteration in recent Go.